packages/mobile/src/i18n/index.ts

```diff
@@ -18,7 +18,9 @@ function getAvailableResources(cachedTranslations: Resource) {
     Object.defineProperty(resources, language, {
       get: () => {
         if (!translation) {
-          translation = _.merge(value!.strings.translation, cachedTranslations[language])
+          translation = __DEV__
+            ? _.merge(cachedTranslations[language], value!.strings.translation)
+            : _.merge(value!.strings.translation, cachedTranslations[language])
```

Comment:
Maybe we should add a small human friendly comment? 🤖 🧑‍🚀

Reply:
teehee good idea, i've added :)